app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -293,7 +294,15 @@ public void toggleLimitedConnectionMode() {
 
     public void centerMapToPlace(Place place) {
         setSelectedItemId(NavTab.NEARBY.code());
-        nearbyParentFragment.centerMapToPlace(place);
```

Comment:
This is probably throwing an exception because the fragment has not yet been attached to the activity, we should probably add that check instead of postDelayed, this does not guarantee solves the issue